src/types/FormTypes.ts

```diff
@@ -66,6 +66,15 @@ export const asSepaInfo = asObject({
 export type SepaInfo = ReturnType<typeof asSepaInfo>
 
 export const ADDRESS_FORM_DISKLET_NAME = 'homeAddress'
+export const EMAIL_CONTACT_FORM_DISKLET_NAME = 'emailContactInfo'
```

Comment:
Should this end with `.json`?

Reply:
I was following the previous pattern, but probably makes sense to just bust the cache for correctness since it's not that big of a loss.